beeflow/cli.py

```diff
@@ -218,14 +218,21 @@ def version_str(version):
     """Convert a version tuple to a string."""
     return '.'.join([str(part) for part in version])
 
+def load_charliecloud():
+    """Load the charliecloud module if it exists."""
+    subprocess.run('module load charliecloud', shell=True, check=True)
```

Comment:
Will this affect the environment of the current process?

Reply:
What happens on a system with no charliecloud module.
Is this persistent?

Reply:
1) The only impact this will have on the current process it that the charliecloud executables will be in it's path if the module exists. 
2) If a system does not contain a charliecloud module, nothing will happen this will just throw an error. 
3) This is not persistent. It only applies to the beeflow process. If the beeflow process is killed such as with beeflow stop, a system restart, or a beeflow restart it will need to be loaded again.